app/src/org/commcare/dalvik/activities/CommCareSetupActivity.java

```diff
@@ -140,7 +146,7 @@
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         CommCareSetupActivity oldActivity = (CommCareSetupActivity)this.getDestroyedActivityState();
-        this.mFromManager = this.getIntent().
+        this.fromManager = this.getIntent().

```

Comment:
I wasn't aware that was a style convention we followed.
My reasoning against the 'm' is that modern IDEs highlight class fields, so there isn't a reason to also label them with an 'm'. Plus we don't seem very good at making sure that variables prefixed by 'm' are actually class fields...

Not prefixing with 'm' does go against Google's style guide for Android platform dev, but I'm fine with that, since we don't work in that codebase.